### PR TITLE
chore: modern rebrand

### PR DESCRIPTION
**Pull request by strophy**
This PR contains simple updates to logos, favicons, colours and fonts to reflect modern Dash branding

### CODE REVIEW

**Review by UdjinM6 — APPROVED** [review]
utACK

File: index.html
Comment (nmarley) [inline index.html]: Probably just my OCD, but the whitespace in the front of the top and bottom lines here is bothering me. 😆

**Review by nmarley — APPROVED** [review]
utACK, colours and images look good 👍

File: index.html
Comment (strophy) [reply index.html]: Good catch, it was indented with tabs instead of spaces

**Comment by strophy** [comment]
Looks like Suba had already started a similar PR on the `rebrand` branch, I've pulled those changes in as well and resolved conflicts.

**Comment by UdjinM6** [comment]
(kind of (un)related: new commits should dismiss stale approvals: Settings -> Branches -> Branch protection rules -> Edit (for master) -> Require a pull request before merging -> Dismiss stale pull request approvals when new commits are pushed)

**Pull request merged**
